src/+datools/+examples/+sandu/l63_experiments.m

```diff
@@ -13,25 +14,25 @@
 variance = 1;
 
 % create an aray of ensemble
-ensNs = [25 50 75 100];
+ensNs = [25, 50, 75, 100];
 
 % create an array of inflation
-infs = [1.01 1.02 1.05 1.10];
+infs = [1.01, 1.02, 1.05, 1.10];
 
 % create an array of rejuvination
-rejs = 2*logspace(-2, -1, 4);
-rejs = round(rejs,2);
+rejs = 2 * logspace(-2, -1, 4);
+rejs = round(rejs, 2);
 
 % define steps and spinups
-spinup = 500;
-times = 11*spinup;
+spinup = 50;
```

Comment:
why did you change this to 50? at least 100 IMO

Reply:
just was making some changes in the plots so wanted something quick. I will revert it to 500 before committing.